docs/application/native/guides/device/diagnostics.md

```diff
@@ -0,0 +1,172 @@
+# Diagnostics API
+
+Diagnostics API is used to collect data from applications and services, and allows to read crash reports if a process in the system crashes.
+
+**Figure: Diagnostics API overall architecture**
+
+![Diagnostics API overall architecture](media/diag-arch.png)
+
+## Prerequisites
+
+1.  To use Diagnostics API to collect diagnostics data, the application must be signed with a platform level distributor certificate.
+
+    To be able to sign application with a platform level certificate:
+
+    1.  Open Certificate Manager in the Tizen Studio which is available in the **Tools** menu.
+    2.  To add a new certificate profile, click **+** in the Certificate Manager.
+
+        ![Add a new profile](media/diag-add-new-profile.png)
+
+    3.  Enter the profile name.
+
+        ![Enter the profile name](media/diag-set-profile-name.png)
+
+    4.  Select the option **Select an existing author certificate**.
+
+        ![Select the option **Select an existing author certificate**](media/diag-select-author-cert.png)
+
+    5.  Choose the **author certificate** from the disk using the **Browse** button.
+
+        ![Choose the author certificate](media/diag-choose-tizen-author-cert.png)
+
+    6.  Select the option **Select a distributor certificate for an another app store**.
+
+        ![Select the option **Select a distributor certificate for an another app store**](media/diag-select-distributor-cert.png)
+
+    7.  Choose the **distributor certificate** from the disk using the **Browse** button.
+
+        ![Choose the distributor certificate](media/diag-choose-tizen-distributor-cert.png)
+
+    8.  Newly added profile should be visible in the profiles list. Close the window by clicking **Close** button.
```

Comment:
This is a step result and not a step. Suggestion:

7.  Choose the **distributor certificate** from the disk using the **Browse** button.
    ![Choose the distributor certificate](media/diag-choose-tizen-distributor-cert.png)
     The newly added profile appears in the profiles list. 

8. Click the **Close** button to close the window.